src/routes/projects.rs

```diff
@@ -146,6 +150,14 @@ pub fn convert_project(
         published: m.published,
         updated: m.updated,
         status: data.status,
+        rejection_data: if let Some(reason) = m.rejection_reason {
```

Comment:
Did modrinth already support rejection reasons in the legacy projects?
  It looks useless if no moderator used it.

Reply:
No